core/src/main/java/com/novoda/noplayer/exoplayer/ExoPlayerCreator.java

```diff
@@ -18,6 +19,7 @@
         this.trackSelector = trackSelector;
     }
 
+    @NonNull
```

Comment:
This is redundant in my opinion. We should assume that everything is non-null unless told otherwise.